azext_iot/central/params.py

```diff
@@ -212,15 +227,40 @@ def load_central_arguments(self, _):
             "by visiting https://github.com/iot-for-all/iot-central-high-availability-clients#readme",
         )
 
+    with self.argument_context("iot central device twin") as context:
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
+            arg_type=get_three_state_flag(),
+            help="The boolean status for replacing device twin, True or False.",
```

Comment:
look at `az iot hub device-identity create` specifically the edge-enabled to see how we format flags.

Reply:
Okay.

Reply:
Right. I was looking at the enabled flag in `az iot central export create`. That help message is not correct for the flag.

Reply:
If we want to use `device twin replace` again, we might need to remove this flag